brouter-core/src/main/java/btools/router/OsmPath.java

```diff
@@ -143,7 +143,7 @@ protected void addAddionalPenalty(OsmTrack refTrack, boolean detailMode, OsmPath
 
     boolean recordTransferNodes = detailMode || rc.countTraffic;
 
-    rc.nogomatch = false;
+    rc.nogoCost = 0.;

```

Comment:
Instead of a boolean `nogomatch`, I'm now storing the cost due to crossing nogos instead. The previous `nogoMatch` can then be inferred by this variable as `Math.abs(rc.nogoCost) > 0`.